ffi/src/util.rs

```diff
@@ -61,3 +62,41 @@ pub fn pubkey(private_key: String) -> PyResult<String> {
     let private_key: StaticSecret = string_to_key(private_key)?;
     Ok(BASE64.encode(PublicKey::from(&private_key).as_bytes()))
 }
+
+/// Convert pem certificate to der certificate and add it to macos keychain.
+#[pyfunction]
+pub fn add_trusted_cert(pem: String) -> PyResult<()> {
+    #[cfg(target_os = "macos")]
+    {
+        let pem_body = pem
+            .lines()
+            .skip(1)
+            .take_while(|&line| line != "-----END CERTIFICATE-----")
+            .collect::<String>();
+        let der = BASE64.decode(pem_body.as_bytes()).unwrap();
+        match macos::add_trusted_cert(der) {
+            Ok(_) => Ok(()),
+            Err(_) => Err(PyErr::new::<PyOSError, _>("Invalid certificate")),
```

Comment:
```suggestion
            Err(e) => Err(PyErr::new::<PyOSError, _>(format!("Failed to install certificate: {:?}", e))),
```

Would something like this be useful to aid debugging bug reports from users? :)